src/SFML/Window/CMakeLists.txt

```
@@ -244,8 +244,8 @@ target_include_directories(sfml-window PRIVATE "${PROJECT_SOURCE_DIR}/extlibs/he
 
 # find and setup usage for external libraries
 if(SFML_OS_LINUX OR SFML_OS_FREEBSD OR SFML_OPENBSD)
-    sfml_find_package(X11 INCLUDE "X11_INCLUDE_DIR" LINK "X11_X11_LIB" "X11_Xrandr_LIB")
-    target_link_libraries(sfml-window PRIVATE X11)
+    sfml_find_package(X11 INCLUDE "X11_INCLUDE_DIR" LINK "X11_X11_LIB" "X11_Xrandr_LIB" "X11_Xcursor_LIB")
+    target_link_libraries(sfml-window PRIVATE X11 Xcursor)
```

```suggestion
    target_link_libraries(sfml-window PRIVATE X11)
```
Remove Xcursor from the line since the Xcursor library was added to the X11 interface target in the line above it. This will fix building on FreeBSD.